advanced/editor-control-identifiers.md

```diff
@@ -90,26 +103,35 @@ Use the [toolbar]({{ site.baseurl }}/configure/editor-appearance/#toolbar) or th
 | fullpage | [fullpage]({{ site.baseurl }}/plugins/fullpage/) | Documents properties for the full page. |
 | ltr | [directionality]({{ site.baseurl }}/plugins/directionality/) | Sets the directionality of contents to ltr. |
 | rtl | [directionality]({{ site.baseurl }}/plugins/directionality/) | Sets the directionality of contents to rtl. |
+| casechange | [casechange]({{ site.baseurl }}/plugins/casechange/) | Changes the case of text in a block selection to uppercase, lowercase, or title case. |
 | emoticons | [emoticons]({{ site.baseurl }}/plugins/emoticons/) | Opens the emoticons dialog. |
 | template | [template]({{ site.baseurl }}/plugins/template/) | Inserts templates into the editor. |
-| forecolor | [textcolor]({{ site.baseurl }}/plugins/textcolor/) | Applies foreground/text color to selection. |
-| backcolor | [textcolor]({{ site.baseurl }}/plugins/textcolor/) | Applies background color to selection. |
 | restoredraft | [restoredraft]({{ site.baseurl }}/plugins/autosave/) | Restores to the latest auto saved draft. |
 | insertfile | [moxiemanager]({{ site.baseurl }}/plugins/moxiemanager/) | Opens the MoxieManager dialog. |
```

Comment:
This should probably be `Tiny Drive` now instead since we're pushing that instead of MoxieManager.

Reply:
yep